### PR TITLE
Formatting changes for RFC2119 key words

### PR DESCRIPTION
**Pull request by MikeRalphson**
This aligns the spec. more closely with the wording and formatting of the OASv3 spec.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,11 +136,11 @@ It combines resource listing and API declaration together into one document.
...
-<a name="A2SInfo"></a>info | [Info Object](#infoObject) | **Required.** Provides metadata about the API. The metadata can be used by the clients if needed.
+<a name="A2SAsyncAPI"></a>asyncapi | [AsyncAPI Version String](#A2SVersionString) | **Required.** Specifies the AsyncAPI Specification version being used. It can be used by tooling Specifications and clients to interpret the version. The structure SHALL be `major`.`minor`.`patch`, where `patch` versions MUST be compatible with the existing `major`.`minor` tooling. Typically patch versions will be introduced to address errors in the documentation, and tooling should typically be compatible with the corresponding `major`.`minor` (1.0.\*). Patch versions will correspond to patches of this document.
```
Comment (fmvilas) [inline README.md]: There is another "Required" at AsyncAPI Version String. Do you think it should also be changed?

Comment (MikeRalphson) [reply README.md]: Yes, it should. Will double check if I've missed any others. Thanks.

Comment (MikeRalphson) [reply README.md]: Updated.

**Comment by fmvilas** [comment]
Thanks @MikeRalphson!

**Pull request merged**